services/go-importer/cmd/assembler/main.go

```diff
@@ -208,6 +283,16 @@ func handlePcap(fname string) {
 		bytes += int64(len(data))
 		done := false
 
+		if !nextFlush.IsZero() {
+			// Check to see if we should flush the streams we have that haven't seen any new data in a while.
+			// Note that pcapOpenOfflineFile is blocking so we need at least see some packets passing by to get here.
+			if time.Now().After(nextFlush) {
```

Comment:
This check opens up a new channel for each iteration which causes additional overhead of opening new channels
```suggestion
			if time.Since(nextFlush) > 0 {
```